libs/milvus/langchain_milvus/vectorstores/milvus.py

```diff
@@ -856,6 +853,82 @@ def add_texts(
             logger.debug("Nothing to insert, skipping.")
             return []
 
+        transposed_embeddings = [
+            [embeddings[j][i] for j in range(len(embeddings))]
+            for i in range(len(embeddings[0]))
+        ]
+        # Now:
+        # transposed_embeddings = [
+        #     [f1(a), f2(a)],
+        #     [f1(b), f2(b)],
+        #     [f1(c), f2(c)]
+        # ]
+
+        return self.add_embeddings(
+            texts=texts,
+            embeddings=transposed_embeddings,
+            metadatas=metadatas,
+            timeout=timeout,
+            batch_size=batch_size,
+            ids=ids,
+            **kwargs,
+        )
```

Comment:
As you can see, at this point, assuming we have `[f1, f2]` embeddings functions and `[a, b, c]` as texts, so:
```
        embeddings = [
            [f1(a), f1(b), f1(c)],
            [f2(a), f2(b), f2(c)]
        ]
```

So what've done is to "transpose" it so we can somewhat have a text and its corresponding list of different vectors (per embeddings function)